apps/web/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+    - name: Install dependencies
+      run: npm ci
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+    - name: Run Playwright tests
+      run: npx playwright test
+    - uses: actions/upload-artifact@v3
+      if: always()
+      with:
+        name: playwright-report
+        path: playwright-report/
+        retention-days: 30
```

Comment:
Assume this is broken now since we're doing things in `apps/web`, I'd recommend making this use turbo to run it in any applicable dirs.

That way we can have a package script like `test:e2e`.

Additionally I think you'll need to bootstrap some devservers, consider using [act](https://github.com/nektos/act) to test locally 😄